[2015-07-10 18:55:24] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /" (from "http://localhost/task/")" at C:\xampp\htdocs\task\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\EventListener\RouterListener.php line 159 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" (from \"http://localhost/task/\") at C:\\xampp\\htdocs\\task\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpKernel\\EventListener\\RouterListener.php:159, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\task\\app\\cache\\prod\\appProdUrlMatcher.php:32)"} []
[2015-07-10 18:56:01] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /new.html"" at C:\xampp\htdocs\task\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\EventListener\RouterListener.php line 159 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /new.html\" at C:\\xampp\\htdocs\\task\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpKernel\\EventListener\\RouterListener.php:159, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\task\\app\\cache\\prod\\appProdUrlMatcher.php:32)"} []
[2015-07-10 18:58:41] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /new.html"" at C:\xampp\htdocs\task\app\cache\prod\classes.php line 2480 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /new.html\" at C:\\xampp\\htdocs\\task\\app\\cache\\prod\\classes.php:2480, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\task\\app\\cache\\prod\\appProdUrlMatcher.php:32)"} []
[2015-07-10 18:58:45] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at C:\xampp\htdocs\task\app\cache\prod\classes.php line 2480 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at C:\\xampp\\htdocs\\task\\app\\cache\\prod\\classes.php:2480, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\task\\app\\cache\\prod\\appProdUrlMatcher.php:32)"} []
[2015-07-10 18:59:37] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at C:\xampp\htdocs\task\app\cache\prod\classes.php line 2480 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at C:\\xampp\\htdocs\\task\\app\\cache\\prod\\classes.php:2480, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\task\\app\\cache\\prod\\appProdUrlMatcher.php:32)"} []
[2015-07-10 18:59:38] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at C:\xampp\htdocs\task\app\cache\prod\classes.php line 2480 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at C:\\xampp\\htdocs\\task\\app\\cache\\prod\\classes.php:2480, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\task\\app\\cache\\prod\\appProdUrlMatcher.php:32)"} []
[2015-07-10 18:59:47] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /index.html"" at C:\xampp\htdocs\task\app\cache\prod\classes.php line 2480 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /index.html\" at C:\\xampp\\htdocs\\task\\app\\cache\\prod\\classes.php:2480, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\task\\app\\cache\\prod\\appProdUrlMatcher.php:32)"} []
[2015-07-10 18:59:54] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at C:\xampp\htdocs\task\app\cache\prod\classes.php line 2480 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at C:\\xampp\\htdocs\\task\\app\\cache\\prod\\classes.php:2480, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\task\\app\\cache\\prod\\appProdUrlMatcher.php:32)"} []
[2015-07-10 19:01:29] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at C:\xampp\htdocs\task\app\cache\prod\classes.php line 2480 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at C:\\xampp\\htdocs\\task\\app\\cache\\prod\\classes.php:2480, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\task\\app\\cache\\prod\\appProdUrlMatcher.php:32)"} []
[2015-07-10 19:01:38] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /app_dev.pl"" at C:\xampp\htdocs\task\app\cache\prod\classes.php line 2480 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /app_dev.pl\" at C:\\xampp\\htdocs\\task\\app\\cache\\prod\\classes.php:2480, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\task\\app\\cache\\prod\\appProdUrlMatcher.php:32)"} []
[2015-07-10 19:02:15] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at C:\xampp\htdocs\task\app\cache\prod\classes.php line 2480 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at C:\\xampp\\htdocs\\task\\app\\cache\\prod\\classes.php:2480, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\task\\app\\cache\\prod\\appProdUrlMatcher.php:32)"} []
[2015-07-10 19:11:04] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at C:\xampp\htdocs\task\app\cache\prod\classes.php line 2480 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at C:\\xampp\\htdocs\\task\\app\\cache\\prod\\classes.php:2480, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\task\\app\\cache\\prod\\appProdUrlMatcher.php:32)"} []
[2015-07-10 19:11:11] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /new.html"" at C:\xampp\htdocs\task\app\cache\prod\classes.php line 2480 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /new.html\" at C:\\xampp\\htdocs\\task\\app\\cache\\prod\\classes.php:2480, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\task\\app\\cache\\prod\\appProdUrlMatcher.php:32)"} []
[2015-07-10 19:18:05] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /new.html"" at C:\xampp\htdocs\task\app\cache\prod\classes.php line 2480 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /new.html\" at C:\\xampp\\htdocs\\task\\app\\cache\\prod\\classes.php:2480, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\task\\app\\cache\\prod\\appProdUrlMatcher.php:32)"} []
[2015-07-10 19:18:06] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /new.html"" at C:\xampp\htdocs\task\app\cache\prod\classes.php line 2480 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /new.html\" at C:\\xampp\\htdocs\\task\\app\\cache\\prod\\classes.php:2480, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\task\\app\\cache\\prod\\appProdUrlMatcher.php:32)"} []
[2015-07-10 19:18:10] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at C:\xampp\htdocs\task\app\cache\prod\classes.php line 2480 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at C:\\xampp\\htdocs\\task\\app\\cache\\prod\\classes.php:2480, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\task\\app\\cache\\prod\\appProdUrlMatcher.php:32)"} []
[2015-07-10 21:20:57] request.INFO: Matched route "_homepage". {"route_parameters":{"_controller":"My\\FormRPBundle\\Controller\\DefaultController::newAction","_route":"_homepage"},"request_uri":"http://localhost/task/web/"} []
[2015-07-10 21:20:57] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2015-07-10 21:20:57] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Form\Exception\InvalidArgumentException: "Could not load type "tesxt"" at C:\xampp\htdocs\task\vendor\symfony\symfony\src\Symfony\Component\Form\FormRegistry.php line 87 {"exception":"[object] (Symfony\\Component\\Form\\Exception\\InvalidArgumentException(code: 0): Could not load type \"tesxt\" at C:\\xampp\\htdocs\\task\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Form\\FormRegistry.php:87)"} []
[2015-07-10 21:20:59] request.INFO: Matched route "_homepage". {"route_parameters":{"_controller":"My\\FormRPBundle\\Controller\\DefaultController::newAction","_route":"_homepage"},"request_uri":"http://localhost/task/web/"} []
[2015-07-10 21:20:59] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2015-07-10 21:20:59] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Form\Exception\InvalidArgumentException: "Could not load type "tesxt"" at C:\xampp\htdocs\task\vendor\symfony\symfony\src\Symfony\Component\Form\FormRegistry.php line 87 {"exception":"[object] (Symfony\\Component\\Form\\Exception\\InvalidArgumentException(code: 0): Could not load type \"tesxt\" at C:\\xampp\\htdocs\\task\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Form\\FormRegistry.php:87)"} []
[2015-07-10 21:51:11] request.INFO: Matched route "_homepage". {"route_parameters":{"_controller":"My\\FormRPBundle\\Controller\\DefaultController::newAction","_route":"_homepage"},"request_uri":"http://localhost/task/web/"} []
[2015-07-10 21:51:11] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2015-07-10 21:51:11] request.CRITICAL: Uncaught PHP Exception LogicException: "The controller must return a response (null given). Did you forget to add a return statement somewhere in your controller?" at C:\xampp\htdocs\task\app\bootstrap.php.cache line 3106 {"exception":"[object] (LogicException(code: 0): The controller must return a response (null given). Did you forget to add a return statement somewhere in your controller? at C:\\xampp\\htdocs\\task\\app\\bootstrap.php.cache:3106)"} []
[2015-07-10 21:51:50] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /success.html"" at C:\xampp\htdocs\task\app\cache\prod\classes.php line 2480 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /success.html\" at C:\\xampp\\htdocs\\task\\app\\cache\\prod\\classes.php:2480, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\task\\app\\cache\\prod\\appProdUrlMatcher.php:39)"} []
[2015-07-11 00:14:21] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /new.html"" at C:\xampp\htdocs\task\app\cache\prod\classes.php line 2480 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /new.html\" at C:\\xampp\\htdocs\\task\\app\\cache\\prod\\classes.php:2480, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\task\\app\\cache\\prod\\appProdUrlMatcher.php:39)"} []
[2015-07-11 00:14:37] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /new.html"" at C:\xampp\htdocs\task\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\EventListener\RouterListener.php line 159 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /new.html\" at C:\\xampp\\htdocs\\task\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\HttpKernel\\EventListener\\RouterListener.php:159, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at C:\\xampp\\htdocs\\task\\app\\cache\\prod\\appProdUrlMatcher.php:39)"} []
[2015-07-11 00:24:25] request.INFO: Matched route "_homepage". {"route_parameters":{"_controller":"My\\FormRPBundle\\Controller\\DefaultController::newAction","_route":"_homepage"},"request_uri":"http://localhost/task/web/"} []
[2015-07-11 00:24:25] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2015-07-11 00:24:26] php.CRITICAL: Fatal Error: Call to undefined method My\FormRPBundle\Entity\Task::isValid() {"type":1,"file":"C:\\xampp\\htdocs\\task\\src\\My\\FormRPBundle\\Controller\\DefaultController.php","line":33,"level":6143,"stack":[{"function":"newAction","type":"->","class":"My\\FormRPBundle\\Controller\\DefaultController","file":"C:\\xampp\\htdocs\\task\\app\\bootstrap.php.cache","line":3094,"args":[]},{"function":"call_user_func_array:{C:\\xampp\\htdocs\\task\\app\\bootstrap.php.cache:3094}","file":"C:\\xampp\\htdocs\\task\\app\\bootstrap.php.cache","line":3094,"args":[]},{"function":"handleRaw","type":"->","class":"Symfony\\Component\\HttpKernel\\HttpKernel","file":"C:\\xampp\\htdocs\\task\\app\\bootstrap.php.cache","line":3056,"args":[]},{"function":"handle","type":"->","class":"Symfony\\Component\\HttpKernel\\HttpKernel","file":"C:\\xampp\\htdocs\\task\\app\\bootstrap.php.cache","line":3207,"args":[]},{"function":"handle","type":"->","class":"Symfony\\Component\\HttpKernel\\DependencyInjection\\ContainerAwareHttpKernel","file":"C:\\xampp\\htdocs\\task\\app\\bootstrap.php.cache","line":2429,"args":[]},{"function":"handle","type":"->","class":"Symfony\\Component\\HttpKernel\\Kernel","file":"C:\\xampp\\htdocs\\task\\web\\app.php","line":28,"args":[]},{"function":"{main}","file":"C:\\xampp\\htdocs\\task\\web\\app.php","line":0,"args":[]}]} []
[2015-07-11 00:24:26] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\UndefinedMethodException: "Attempted to call an undefined method named "isValid" of class "My\FormRPBundle\Entity\Task"." at C:\xampp\htdocs\task\src\My\FormRPBundle\Controller\DefaultController.php line 33 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\UndefinedMethodException(code: 0): Attempted to call an undefined method named \"isValid\" of class \"My\\FormRPBundle\\Entity\\Task\". at C:\\xampp\\htdocs\\task\\src\\My\\FormRPBundle\\Controller\\DefaultController.php:33)"} []
[2015-07-11 00:30:43] request.INFO: Matched route "_homepage". {"route_parameters":{"_controller":"My\\FormRPBundle\\Controller\\DefaultController::newAction","_route":"_homepage"},"request_uri":"http://localhost/task/web/"} []
[2015-07-11 00:30:43] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2015-07-11 00:30:44] request.CRITICAL: Uncaught PHP Exception LogicException: "The controller must return a response (null given). Did you forget to add a return statement somewhere in your controller?" at C:\xampp\htdocs\task\app\bootstrap.php.cache line 3106 {"exception":"[object] (LogicException(code: 0): The controller must return a response (null given). Did you forget to add a return statement somewhere in your controller? at C:\\xampp\\htdocs\\task\\app\\bootstrap.php.cache:3106)"} []
[2015-07-11 00:50:00] request.INFO: Matched route "_homepage". {"route_parameters":{"_controller":"My\\FormRPBundle\\Controller\\DefaultController::newAction","_route":"_homepage"},"request_uri":"http://localhost/task/web/"} []
[2015-07-11 00:50:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2015-07-11 00:50:00] php.CRITICAL: Fatal Parse Error: syntax error, unexpected '}', expecting ')' {"type":4,"file":"C:\\xampp\\htdocs\\task\\src\\My\\FormRPBundle\\Controller\\DefaultController.php","line":46,"level":6143,"stack":[{"function":"Composer\\Autoload\\includeFile","file":"C:\\xampp\\htdocs\\task\\vendor\\composer\\ClassLoader.php","line":301,"args":[]},{"function":"loadClass","type":"->","class":"Composer\\Autoload\\ClassLoader","file":"C:\\xampp\\htdocs\\task\\app\\cache\\prod\\classes.php","line":0,"args":[]},{"function":"spl_autoload_call","file":"C:\\xampp\\htdocs\\task\\app\\cache\\prod\\classes.php","line":2922,"args":[]},{"function":"class_exists","file":"C:\\xampp\\htdocs\\task\\app\\cache\\prod\\classes.php","line":2922,"args":[]},{"function":"createController","type":"->","class":"Symfony\\Bundle\\FrameworkBundle\\Controller\\ControllerResolver","file":"C:\\xampp\\htdocs\\task\\app\\cache\\prod\\classes.php","line":2539,"args":[]},{"function":"getController","type":"->","class":"Symfony\\Component\\HttpKernel\\Controller\\ControllerResolver","file":"C:\\xampp\\htdocs\\task\\app\\bootstrap.php.cache","line":3087,"args":[]},{"function":"handleRaw","type":"->","class":"Symfony\\Component\\HttpKernel\\HttpKernel","file":"C:\\xampp\\htdocs\\task\\app\\bootstrap.php.cache","line":3056,"args":[]},{"function":"handle","type":"->","class":"Symfony\\Component\\HttpKernel\\HttpKernel","file":"C:\\xampp\\htdocs\\task\\app\\bootstrap.php.cache","line":3207,"args":[]},{"function":"handle","type":"->","class":"Symfony\\Component\\HttpKernel\\DependencyInjection\\ContainerAwareHttpKernel","file":"C:\\xampp\\htdocs\\task\\app\\bootstrap.php.cache","line":2429,"args":[]},{"function":"handle","type":"->","class":"Symfony\\Component\\HttpKernel\\Kernel","file":"C:\\xampp\\htdocs\\task\\web\\app.php","line":28,"args":[]},{"function":"{main}","file":"C:\\xampp\\htdocs\\task\\web\\app.php","line":0,"args":[]}]} []
[2015-07-11 00:50:00] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Parse Error: syntax error, unexpected '}', expecting ')'" at C:\xampp\htdocs\task\src\My\FormRPBundle\Controller\DefaultController.php line 46 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Parse Error: syntax error, unexpected '}', expecting ')' at C:\\xampp\\htdocs\\task\\src\\My\\FormRPBundle\\Controller\\DefaultController.php:46)"} []
[2015-07-11 00:52:05] request.INFO: Matched route "_homepage". {"route_parameters":{"_controller":"My\\FormRPBundle\\Controller\\DefaultController::newAction","_route":"_homepage"},"request_uri":"http://localhost/task/web/"} []
[2015-07-11 00:52:05] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2015-07-11 00:52:05] php.CRITICAL: Fatal Error: Call to a member function handleRequest() on null {"type":1,"file":"C:\\xampp\\htdocs\\task\\src\\My\\FormRPBundle\\Controller\\DefaultController.php","line":21,"level":6143,"stack":[{"function":"newAction","type":"->","class":"My\\FormRPBundle\\Controller\\DefaultController","file":"C:\\xampp\\htdocs\\task\\app\\bootstrap.php.cache","line":3094,"args":[]},{"function":"call_user_func_array:{C:\\xampp\\htdocs\\task\\app\\bootstrap.php.cache:3094}","file":"C:\\xampp\\htdocs\\task\\app\\bootstrap.php.cache","line":3094,"args":[]},{"function":"handleRaw","type":"->","class":"Symfony\\Component\\HttpKernel\\HttpKernel","file":"C:\\xampp\\htdocs\\task\\app\\bootstrap.php.cache","line":3056,"args":[]},{"function":"handle","type":"->","class":"Symfony\\Component\\HttpKernel\\HttpKernel","file":"C:\\xampp\\htdocs\\task\\app\\bootstrap.php.cache","line":3207,"args":[]},{"function":"handle","type":"->","class":"Symfony\\Component\\HttpKernel\\DependencyInjection\\ContainerAwareHttpKernel","file":"C:\\xampp\\htdocs\\task\\app\\bootstrap.php.cache","line":2429,"args":[]},{"function":"handle","type":"->","class":"Symfony\\Component\\HttpKernel\\Kernel","file":"C:\\xampp\\htdocs\\task\\web\\app.php","line":28,"args":[]},{"function":"{main}","file":"C:\\xampp\\htdocs\\task\\web\\app.php","line":0,"args":[]}]} []
[2015-07-11 00:52:05] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Error: Call to a member function handleRequest() on null" at C:\xampp\htdocs\task\src\My\FormRPBundle\Controller\DefaultController.php line 21 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Error: Call to a member function handleRequest() on null at C:\\xampp\\htdocs\\task\\src\\My\\FormRPBundle\\Controller\\DefaultController.php:21)"} []
[2015-07-11 00:52:37] request.INFO: Matched route "_homepage". {"route_parameters":{"_controller":"My\\FormRPBundle\\Controller\\DefaultController::newAction","_route":"_homepage"},"request_uri":"http://localhost/task/web/"} []
[2015-07-11 00:52:37] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2015-07-11 00:52:37] request.CRITICAL: Uncaught PHP Exception Symfony\Component\PropertyAccess\Exception\NoSuchPropertyException: "Neither the property "emaila" nor one of the methods "getEmaila()", "emaila()", "isEmaila()", "hasEmaila()", "__get()" exist and have public access in class "My\FormRPBundle\Entity\Task"." at C:\xampp\htdocs\task\vendor\symfony\symfony\src\Symfony\Component\PropertyAccess\PropertyAccessor.php line 339 {"exception":"[object] (Symfony\\Component\\PropertyAccess\\Exception\\NoSuchPropertyException(code: 0): Neither the property \"emaila\" nor one of the methods \"getEmaila()\", \"emaila()\", \"isEmaila()\", \"hasEmaila()\", \"__get()\" exist and have public access in class \"My\\FormRPBundle\\Entity\\Task\". at C:\\xampp\\htdocs\\task\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\PropertyAccess\\PropertyAccessor.php:339)"} []
[2015-07-11 01:18:17] request.INFO: Matched route "_homepage". {"route_parameters":{"_controller":"My\\FormRPBundle\\Controller\\DefaultController::newAction","_route":"_homepage"},"request_uri":"http://localhost/task/web/"} []
[2015-07-11 01:18:17] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2015-07-11 01:18:17] request.CRITICAL: Uncaught PHP Exception Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException: "You have requested a non-existent service "reuquest". Did you mean this: "request"?" at C:\xampp\htdocs\task\app\bootstrap.php.cache line 2134 {"exception":"[object] (Symfony\\Component\\DependencyInjection\\Exception\\ServiceNotFoundException(code: 0): You have requested a non-existent service \"reuquest\". Did you mean this: \"request\"? at C:\\xampp\\htdocs\\task\\app\\bootstrap.php.cache:2134)"} []
[2015-07-11 01:18:37] request.INFO: Matched route "_homepage". {"route_parameters":{"_controller":"My\\FormRPBundle\\Controller\\DefaultController::newAction","_route":"_homepage"},"request_uri":"http://localhost/task/web/"} []
[2015-07-11 01:18:37] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2015-07-11 01:18:37] request.CRITICAL: Uncaught PHP Exception Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException: "You have requested a non-existent service "reuquest". Did you mean this: "request"?" at C:\xampp\htdocs\task\app\bootstrap.php.cache line 2134 {"exception":"[object] (Symfony\\Component\\DependencyInjection\\Exception\\ServiceNotFoundException(code: 0): You have requested a non-existent service \"reuquest\". Did you mean this: \"request\"? at C:\\xampp\\htdocs\\task\\app\\bootstrap.php.cache:2134)"} []
[2015-07-11 01:32:31] request.INFO: Matched route "_homepage". {"route_parameters":{"_controller":"My\\FormRPBundle\\Controller\\DefaultController::newAction","_route":"_homepage"},"request_uri":"http://localhost/task/web/"} []
[2015-07-11 01:32:31] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2015-07-11 01:32:31] request.CRITICAL: Uncaught PHP Exception Twig_Error_Runtime: "An exception has been thrown during the rendering of a template ("Catchable Fatal Error: Argument 1 passed to Symfony\Component\Form\FormRenderer::renderBlock() must be an instance of Symfony\Component\Form\FormView, null given, called in C:\xampp\htdocs\task\app\cache\prod\twig\1\2\12f34e394112d59e7efb6a3dd024a20a15f0155471bac0d12b8e3f1a48a3badc.php on line 24 and defined") in "MyFormRPBundle:Default:new.html.twig" at line 3." at C:\xampp\htdocs\task\app\cache\prod\classes.php line 5173 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Catchable Fatal Error: Argument 1 passed to Symfony\\Component\\Form\\FormRenderer::renderBlock() must be an instance of Symfony\\Component\\Form\\FormView, null given, called in C:\\xampp\\htdocs\\task\\app\\cache\\prod\\twig\\1\\2\\12f34e394112d59e7efb6a3dd024a20a15f0155471bac0d12b8e3f1a48a3badc.php on line 24 and defined\") in \"MyFormRPBundle:Default:new.html.twig\" at line 3. at C:\\xampp\\htdocs\\task\\app\\cache\\prod\\classes.php:5173, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 1 passed to Symfony\\Component\\Form\\FormRenderer::renderBlock() must be an instance of Symfony\\Component\\Form\\FormView, null given, called in C:\\xampp\\htdocs\\task\\app\\cache\\prod\\twig\\1\\2\\12f34e394112d59e7efb6a3dd024a20a15f0155471bac0d12b8e3f1a48a3badc.php on line 24 and defined at C:\\xampp\\htdocs\\task\\vendor\\symfony\\symfony\\src\\Symfony\\Component\\Form\\FormRenderer.php:106)"} []
[2015-07-11 15:23:22] request.INFO: Matched route "_homepage". {"route_parameters":{"_controller":"My\\FormRPBundle\\Controller\\DefaultController::newAction","_route":"_homepage"},"request_uri":"http://localhost/task/web/"} []
[2015-07-11 15:23:22] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2015-07-11 15:23:24] request.CRITICAL: Uncaught PHP Exception Twig_Error_Runtime: "An exception has been thrown during the rendering of a template ("Catchable Fatal Error: Object of class DateTime could not be converted to string") in "MyFormRPBundle:Default:success.html.twig" at line 2." at C:\xampp\htdocs\task\vendor\twig\twig\lib\Twig\Template.php line 337 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Catchable Fatal Error: Object of class DateTime could not be converted to string\") in \"MyFormRPBundle:Default:success.html.twig\" at line 2. at C:\\xampp\\htdocs\\task\\vendor\\twig\\twig\\lib\\Twig\\Template.php:337, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Object of class DateTime could not be converted to string at C:\\xampp\\htdocs\\task\\app\\cache\\prod\\twig\\e\\b\\eb487b0e58b34c33873ab63267710e1f62618ef1bc223f201c9cae7c07dc3476.php:28)"} []
[2015-07-11 15:23:56] request.INFO: Matched route "_homepage". {"route_parameters":{"_controller":"My\\FormRPBundle\\Controller\\DefaultController::newAction","_route":"_homepage"},"request_uri":"http://localhost/task/web/"} []
[2015-07-11 15:23:56] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2015-07-11 15:23:56] request.CRITICAL: Uncaught PHP Exception Twig_Error_Runtime: "An exception has been thrown during the rendering of a template ("Catchable Fatal Error: Object of class DateTime could not be converted to string") in "MyFormRPBundle:Default:success.html.twig" at line 2." at C:\xampp\htdocs\task\vendor\twig\twig\lib\Twig\Template.php line 337 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Catchable Fatal Error: Object of class DateTime could not be converted to string\") in \"MyFormRPBundle:Default:success.html.twig\" at line 2. at C:\\xampp\\htdocs\\task\\vendor\\twig\\twig\\lib\\Twig\\Template.php:337, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Object of class DateTime could not be converted to string at C:\\xampp\\htdocs\\task\\app\\cache\\prod\\twig\\e\\b\\eb487b0e58b34c33873ab63267710e1f62618ef1bc223f201c9cae7c07dc3476.php:28)"} []
[2015-07-11 15:24:00] request.INFO: Matched route "_homepage". {"route_parameters":{"_controller":"My\\FormRPBundle\\Controller\\DefaultController::newAction","_route":"_homepage"},"request_uri":"http://localhost/task/web/"} []
[2015-07-11 15:24:00] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2015-07-11 15:24:00] request.CRITICAL: Uncaught PHP Exception Twig_Error_Runtime: "An exception has been thrown during the rendering of a template ("Catchable Fatal Error: Object of class DateTime could not be converted to string") in "MyFormRPBundle:Default:success.html.twig" at line 2." at C:\xampp\htdocs\task\app\cache\prod\classes.php line 5173 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Catchable Fatal Error: Object of class DateTime could not be converted to string\") in \"MyFormRPBundle:Default:success.html.twig\" at line 2. at C:\\xampp\\htdocs\\task\\app\\cache\\prod\\classes.php:5173, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Object of class DateTime could not be converted to string at C:\\xampp\\htdocs\\task\\app\\cache\\prod\\twig\\e\\b\\eb487b0e58b34c33873ab63267710e1f62618ef1bc223f201c9cae7c07dc3476.php:28)"} []
[2015-07-12 23:50:03] request.INFO: Matched route "_homepage". {"route_parameters":{"_controller":"My\\FormRPBundle\\Controller\\DefaultController::newAction","_route":"_homepage"},"request_uri":"http://localhost/task/web/"} []
[2015-07-12 23:50:03] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2015-07-12 23:50:04] php.CRITICAL: Fatal Error: Call to undefined function DateTime() {"type":1,"file":"C:\\xampp\\htdocs\\task\\src\\My\\FormRPBundle\\Controller\\DefaultController.php","line":33,"level":6143,"stack":[{"function":"newAction","type":"->","class":"My\\FormRPBundle\\Controller\\DefaultController","file":"C:\\xampp\\htdocs\\task\\app\\bootstrap.php.cache","line":3094,"args":[]},{"function":"call_user_func_array:{C:\\xampp\\htdocs\\task\\app\\bootstrap.php.cache:3094}","file":"C:\\xampp\\htdocs\\task\\app\\bootstrap.php.cache","line":3094,"args":[]},{"function":"handleRaw","type":"->","class":"Symfony\\Component\\HttpKernel\\HttpKernel","file":"C:\\xampp\\htdocs\\task\\app\\bootstrap.php.cache","line":3056,"args":[]},{"function":"handle","type":"->","class":"Symfony\\Component\\HttpKernel\\HttpKernel","file":"C:\\xampp\\htdocs\\task\\app\\bootstrap.php.cache","line":3207,"args":[]},{"function":"handle","type":"->","class":"Symfony\\Component\\HttpKernel\\DependencyInjection\\ContainerAwareHttpKernel","file":"C:\\xampp\\htdocs\\task\\app\\bootstrap.php.cache","line":2429,"args":[]},{"function":"handle","type":"->","class":"Symfony\\Component\\HttpKernel\\Kernel","file":"C:\\xampp\\htdocs\\task\\web\\app.php","line":28,"args":[]},{"function":"{main}","file":"C:\\xampp\\htdocs\\task\\web\\app.php","line":0,"args":[]}]} []
[2015-07-12 23:50:04] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\UndefinedFunctionException: "Attempted to call function "DateTime" from the global namespace." at C:\xampp\htdocs\task\src\My\FormRPBundle\Controller\DefaultController.php line 33 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\UndefinedFunctionException(code: 0): Attempted to call function \"DateTime\" from the global namespace. at C:\\xampp\\htdocs\\task\\src\\My\\FormRPBundle\\Controller\\DefaultController.php:33)"} []
[2015-07-12 23:50:33] request.INFO: Matched route "_homepage". {"route_parameters":{"_controller":"My\\FormRPBundle\\Controller\\DefaultController::newAction","_route":"_homepage"},"request_uri":"http://localhost/task/web/"} []
[2015-07-12 23:50:33] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2015-07-12 23:50:33] php.CRITICAL: Fatal Error: Call to undefined function My\FormRPBundle\Controller\DateTime() {"type":1,"file":"C:\\xampp\\htdocs\\task\\src\\My\\FormRPBundle\\Controller\\DefaultController.php","line":33,"level":6143,"stack":[{"function":"newAction","type":"->","class":"My\\FormRPBundle\\Controller\\DefaultController","file":"C:\\xampp\\htdocs\\task\\app\\bootstrap.php.cache","line":3094,"args":[]},{"function":"call_user_func_array:{C:\\xampp\\htdocs\\task\\app\\bootstrap.php.cache:3094}","file":"C:\\xampp\\htdocs\\task\\app\\bootstrap.php.cache","line":3094,"args":[]},{"function":"handleRaw","type":"->","class":"Symfony\\Component\\HttpKernel\\HttpKernel","file":"C:\\xampp\\htdocs\\task\\app\\bootstrap.php.cache","line":3056,"args":[]},{"function":"handle","type":"->","class":"Symfony\\Component\\HttpKernel\\HttpKernel","file":"C:\\xampp\\htdocs\\task\\app\\bootstrap.php.cache","line":3207,"args":[]},{"function":"handle","type":"->","class":"Symfony\\Component\\HttpKernel\\DependencyInjection\\ContainerAwareHttpKernel","file":"C:\\xampp\\htdocs\\task\\app\\bootstrap.php.cache","line":2429,"args":[]},{"function":"handle","type":"->","class":"Symfony\\Component\\HttpKernel\\Kernel","file":"C:\\xampp\\htdocs\\task\\web\\app.php","line":28,"args":[]},{"function":"{main}","file":"C:\\xampp\\htdocs\\task\\web\\app.php","line":0,"args":[]}]} []
[2015-07-12 23:50:33] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\UndefinedFunctionException: "Attempted to call function "DateTime" from namespace "My\FormRPBundle\Controller"." at C:\xampp\htdocs\task\src\My\FormRPBundle\Controller\DefaultController.php line 33 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\UndefinedFunctionException(code: 0): Attempted to call function \"DateTime\" from namespace \"My\\FormRPBundle\\Controller\". at C:\\xampp\\htdocs\\task\\src\\My\\FormRPBundle\\Controller\\DefaultController.php:33)"} []
[2015-07-12 23:50:49] request.INFO: Matched route "_homepage". {"route_parameters":{"_controller":"My\\FormRPBundle\\Controller\\DefaultController::newAction","_route":"_homepage"},"request_uri":"http://localhost/task/web/"} []
[2015-07-12 23:50:49] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2015-07-12 23:50:50] request.CRITICAL: Uncaught PHP Exception Twig_Error_Runtime: "An exception has been thrown during the rendering of a template ("Catchable Fatal Error: Object of class DateInterval could not be converted to string") in "MyFormRPBundle:Default:success.html.twig" at line 2." at C:\xampp\htdocs\task\app\cache\prod\classes.php line 5173 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Catchable Fatal Error: Object of class DateInterval could not be converted to string\") in \"MyFormRPBundle:Default:success.html.twig\" at line 2. at C:\\xampp\\htdocs\\task\\app\\cache\\prod\\classes.php:5173, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Object of class DateInterval could not be converted to string at C:\\xampp\\htdocs\\task\\app\\cache\\prod\\twig\\e\\b\\eb487b0e58b34c33873ab63267710e1f62618ef1bc223f201c9cae7c07dc3476.php:28)"} []
[2015-07-14 00:34:26] request.INFO: Matched route "_homepage". {"route_parameters":{"_controller":"My\\FormRPBundle\\Controller\\DefaultController::newAction","_route":"_homepage"},"request_uri":"http://localhost/task/web/"} []
[2015-07-14 00:34:26] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2015-07-14 00:34:27] request.CRITICAL: Uncaught PHP Exception Doctrine\Common\Annotations\AnnotationException: "[Semantical Error] The annotation "@My\FormRPBundle\Validator\Constraint\DateExam" in property My\FormRPBundle\Entity\Task::$birth does not exist, or could not be auto-loaded." at C:\xampp\htdocs\task\vendor\doctrine\annotations\lib\Doctrine\Common\Annotations\AnnotationException.php line 54 {"exception":"[object] (Doctrine\\Common\\Annotations\\AnnotationException(code: 0): [Semantical Error] The annotation \"@My\\FormRPBundle\\Validator\\Constraint\\DateExam\" in property My\\FormRPBundle\\Entity\\Task::$birth does not exist, or could not be auto-loaded. at C:\\xampp\\htdocs\\task\\vendor\\doctrine\\annotations\\lib\\Doctrine\\Common\\Annotations\\AnnotationException.php:54)"} []
[2015-07-14 00:34:59] request.INFO: Matched route "_homepage". {"route_parameters":{"_controller":"My\\FormRPBundle\\Controller\\DefaultController::newAction","_route":"_homepage"},"request_uri":"http://localhost/task/web/"} []
[2015-07-14 00:34:59] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2015-07-14 00:35:00] request.CRITICAL: Uncaught PHP Exception Doctrine\Common\Annotations\AnnotationException: "[Semantical Error] The annotation "@My\FormRPBundle\Validator\Constraint\DateExam" in property My\FormRPBundle\Entity\Task::$birth does not exist, or could not be auto-loaded." at C:\xampp\htdocs\task\vendor\doctrine\annotations\lib\Doctrine\Common\Annotations\AnnotationException.php line 54 {"exception":"[object] (Doctrine\\Common\\Annotations\\AnnotationException(code: 0): [Semantical Error] The annotation \"@My\\FormRPBundle\\Validator\\Constraint\\DateExam\" in property My\\FormRPBundle\\Entity\\Task::$birth does not exist, or could not be auto-loaded. at C:\\xampp\\htdocs\\task\\vendor\\doctrine\\annotations\\lib\\Doctrine\\Common\\Annotations\\AnnotationException.php:54)"} []
[2015-07-14 22:45:49] request.INFO: Matched route "_homepage". {"route_parameters":{"_controller":"My\\FormRPBundle\\Controller\\DefaultController::newAction","_route":"_homepage"},"request_uri":"http://localhost/task/web/"} []
[2015-07-14 22:45:49] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2015-07-14 22:45:51] request.CRITICAL: Uncaught PHP Exception Doctrine\Common\Annotations\AnnotationException: "[Semantical Error] The annotation "@My\FormRPBundle\Validator\Constraint\DateExam" in property My\FormRPBundle\Entity\Task::$birth does not exist, or could not be auto-loaded." at C:\xampp\htdocs\task\vendor\doctrine\annotations\lib\Doctrine\Common\Annotations\AnnotationException.php line 54 {"exception":"[object] (Doctrine\\Common\\Annotations\\AnnotationException(code: 0): [Semantical Error] The annotation \"@My\\FormRPBundle\\Validator\\Constraint\\DateExam\" in property My\\FormRPBundle\\Entity\\Task::$birth does not exist, or could not be auto-loaded. at C:\\xampp\\htdocs\\task\\vendor\\doctrine\\annotations\\lib\\Doctrine\\Common\\Annotations\\AnnotationException.php:54)"} []
